draft-ietf-quic-transport.md

```diff
@@ -3900,74 +3903,46 @@ max_packet_size (0x0005):
   Values below 1200 are invalid.  This limit only applies to protected packets
   ({{packet-protected}}).
 
-ack_delay_exponent (0x0007):
-
-: An 8-bit unsigned integer value indicating an exponent used to decode the ACK
-  Delay field in the ACK frame, see {{frame-ack}}.  If this value is absent, a
-  default value of 3 is assumed (indicating a multiplier of 8).  The default
-  value is also used for ACK frames that are sent in Initial and Handshake
-  packets.  Values above 20 are invalid.
-
-disable_migration (0x0009):
-
-: The endpoint does not support connection migration ({{migration}}). Peers MUST
-  NOT send any packets, including probing packets ({{probing}}), from a local
-  address other than that used to perform the handshake.  This parameter is a
-  zero-length value.
+initial_max_data (0x0004):
 
-max_ack_delay (0x000c):
-
-: An 8 bit unsigned integer value indicating the maximum amount of time in
-  milliseconds by which the endpoint will delay sending acknowledgments.
-  If this value is absent, a default of 25 milliseconds is assumed.
-
-Either peer MAY advertise an initial value for flow control of each type of
-stream on which they might receive data.  Each of the following transport
-parameters is encoded as an unsigned 32-bit integer in units of bytes:
+: The initial maximum data parameter contains the initial value for the maximum
+  amount of data that can be sent on the connection.  This parameter is encoded
+  as an unsigned 32-bit integer in units of bytes.  This is equivalent to
+  sending a MAX_DATA ({{frame-max-data}}) for the connection immediately after
+  completing the handshake. If the transport parameter is absent, the connection
+  starts with a flow control limit of 0.
 
-initial_max_stream_data_bidi_local (0x0000):
+initial_max_stream_data_bidi_local (0x0005):
 
 : The initial stream maximum data for bidirectional, locally-initiated streams
```

Comment:
```suggestion
: The parameter specifying the initial limit on stream data for bidirectional, locally-initiated
  streams is a 32-bit integer that contains the initial flow control limit for
```

Reply:
This parameter is a 32-bit integer specifying the initial flow control limit for locally-initiated bidirectional streams. This limit applies to newly created bidirectional streams opened by the endpoint that sends the transport parameter.